runtimes/polimec/src/lib.rs

```diff
@@ -429,9 +429,35 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 parameter_types! {
 	pub const RelayOrigin: AggregateMessageOrigin = AggregateMessageOrigin::Parent;
 }
+pub struct MockedChannelInfo;
```

Comment:
Can you add a todo comment to this struct why we need it and when we can remove it?
Might also be good to make a linear issue for all TODOs we need to do for next version upgrade. Haven't done it myself but just though of it.